### ISSUE TITLE
Movie Test Failed

### ISSUE THREAD

**Issue by GilTorch** [myflashlab/facebook-ANE#83]
Hi, I am try to impelement Facebook Connect on Animate CC. Here's the issue I've encountered:
The content cannot be loaded because there was a problem loading an extension: Error: Extension namespace is invalid for C:\Users\GILBER~1\AppData\Local\Temp\Tmp_ANE_File_Unzipped_Packages\facebook.ane

**Comment by myflashlab** [comment]
When you see the ```namespace is invalid``` error message, it means that you must use the latest AIR SDK, for example try AIR 25 and then make sure you are modifying your manifest .xml file and set it to ```<application xmlns="http://ns.adobe.com/air/application/25.0">```

**Comment by myflashlab** [comment]
could you update us about this issue?

**Comment by GilTorch** [comment]
All my problems were in the XML file and after that it gave me an error
because I tested it on the Desktop Emulator. I tested on a Mobile Device
and took the Hash Key then I put it on the app on Facebook for developeres
and now it is working just fine.  I wish every ANE you created had a
complete video tutorial.

Torchon Gilbert
Ingénieur Informaticien, Graphiste 3D-2D, Programmeur, Membre du
Laboratoire de Réalite Virtuelle de l'ESIH
No. Téléphone: +509 48550669

2017-06-20 5:06 GMT-05:00 MyFlashLabs <notifications@github.com>:
